README.md

```diff
@@ -108,16 +108,31 @@ When the upload is completed, click the **Data sets** tab and view the details o
 1. In the TAP console, navigate to the **Data Science > Jupyter**. Click the **Create a new Jupyter instance** button.
 1. Copy the password for the newly created Jupyter instance (for use in the next step).
 1. Login to Jupyter by clicking the App Url link, using the password you just copied.
-
-#### Install TAP Analytics Toolkit client
-1. Create a new Jupyter notebook (see the steps above).
-1. Install the TAP Analytics Toolkit client in your notebook by executing this command: ‘!pip install <link-to-atk-server>/client’. <link-to-atk-server> can be copied from the URL column in the **TAP Analytics Toolkit** instances list.
-
-#### Connect to TAP Analytics Toolkit server and run model generation script
-1. Copy the contents from ‘src/main/atkmodelgenerator/atk_model_generator.py’ into your notebook.
-1. After the imports section, set the URI to the TAP Analytics Toolkit server: ‘ta.server.uri = <link-to-atk-server>’
-1. Set the value of ‘ds’ to be the link to the data set previously uploaded to HDFS (‘targetUri’).
-1. Run the script. The link to the created model in HDFS will be printed in the output.
+1. In Jupyter, click the (white) **Upload** button, then navigate to the Space Shuttle notebook file, which can be found here: `space-shuttle-demo-master/sparktkmodelgenerator/sparktk_shuttle_model_generator.ipynb`
+1. Select the file and click the second (blue) **Upload** button.
+1. In Jupyter, click on the `sparktk_shuttle_model_generator.ipynb` file to open the file.
+
+#### Connecting to TAP server and generating the model
+In this step, you modify the generic notebook script, then run it to create the model.
```

Comment:
First two steps in this sequence are not rendering. Usually if I add two spaces at the end of the previous text, that fixes the rendering.